accounts/models.py

```diff
@@ -7,11 +7,19 @@
 class User(AbstractUser):
     username = None
     email = models.EmailField(unique=True)
+    firebase_uid = models.CharField(max_length=128)
 
     USERNAME_FIELD = "email"
     REQUIRED_FIELDS = []
 
     objects = UserManager()
 
+    class Meta:
+        indexes = [models.Index(fields=["firebase_uid"])]
+
     def __str__(self):
         return self.email
+
+    @property
+    def is_authenticated(self):
```

Comment:
How does this know if the user is authenticated?

Reply:
that code is actually legacy from somethhing i was trying out before :o good catch haha, i'm deleting this